TestFoundation/TestNSProcessInfo.swift

```diff
@@ -0,0 +1,35 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+
+#if DEPLOYMENT_RUNTIME_OBJC || os(Linux)
+import Foundation
+import XCTest
+#else
+import SwiftFoundation
+import SwiftXCTest
+#endif
+
+
+class TestNSProcessInfo : XCTestCase {
+    
+    var allTests : [(String, () -> ())] {
+        return [
+            ("test_operatingSystemVersion", test_operatingSystemVersion ),
+        ]
+    }
+    
+    func test_operatingSystemVersion() {
+        let processInfo = NSProcessInfo.processInfo()
+        let versionString = processInfo.operatingSystemVersionString
+        let version = processInfo.operatingSystemVersion
+        let reconstructedVersionString = "Version \(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+        XCTAssert(versionString.hasPrefix(reconstructedVersionString))
```

Comment:
I think the best approach for this test is to just verify that the 3 strings are non-zero length (each).